config/module.config.php

```diff
@@ -38,11 +38,12 @@
         ],
     ],
     'service_manager' => [
-        'invokables' => [
-            'BsbFlysystemManager' => 'BsbFlysystem\Service\FilesystemManager',
+        'aliases' => [
+            'BsbFlysystem\Service\FilesystemManager' => 'BsbFlysystemManager',
```

Comment:
this maybe not really required as it is not documented, but many people use `BsbFlysystem\Service\FilesystemManager::class` syntax